CONTRIBUTING.md

```diff
@@ -0,0 +1,94 @@
+# HERMES development workflow
+
+## Preamble
+
+> **Branching is cheap!**
+>
+> Work in small packets, put up small pull requests!
+>
+> Aim for quick turnaround times!
+
+## Branching
+
+We loosely follow a mixture of [GitHubFlow](https://docs.github.com/en/get-started/quickstart/github-flow) and [GitFlow](https://nvie.com/posts/a-successful-git-branching-model/):
+
+`main`
+    - Stable branch
+    - Merges come only from `develop` or a hotfix branch (i.e., when something needs to be fixed in "production")
```

Comment:
Allowing only hotfixes to go into main directly is sensible. This line suggests though, that `hotfix` is a special type of branch. Below, `feature/` branches are defined specifically. To complement this rule, maybe we should also define a `hotfix/` branch.

Reply:
Thanks, good point, I'll have a look.

Reply:
@jkelling Added a poitn re `hotfix` branches in b62068ec299e0e86d66ff0c4b15cc792fdf7d5b7.